scripts/install-linters.sh

```diff
@@ -1,29 +1,43 @@
-sudo add-apt-repository --yes ppa:kalakris/cmake
+set -x
+set -e
+
+root="$(git rev-parse --show-toplevel)"
+${root}/scripts/clean-linters.sh
```

Comment:
This was in the first place to get the `install-linters.sh` running with `set -e` when I incrementally tried to get passed each line. Since `git clone https://github.com/htacg/tidy-html5.git` will complain if there is already a directory. And also there is some stuff `pmd` stuff lying around
![screenshot from 2017-08-13 22-25-23](https://user-images.githubusercontent.com/17890112/29253158-60ba2c9c-8076-11e7-9365-b1fa5ce4b0a1.png)
Just thought its cleaner to remove this stuff when you do a rerun because some line failed.